Formula/earthly.rb

```diff
@@ -16,8 +16,8 @@ class Earthly < Formula
   depends_on "go" => :build
 
   def install
-    ldflags = "-X main.DefaultBuildkitdImage=earthly/buildkitd:v0.3.5 -X main.Version=v0.3.5 -X" \
-              " main.GitSha=557634495eb6cf9d48d05bc25461f34b2f86f06f "
+    ldflags = "-X main.DefaultBuildkitdImage=earthly/buildkitd:v0.3.6 -X main.Version=v0.3.6 -X" \
```

Comment:
This should just use `#{version}`

Reply:
thanks for pointing that out @SMillerDev , how does `version` get set?

Reply:
I think it is automatically inferred from the URL. Though I think it doesn't contain the `v`.

@SMillerDev - we are actually automatically generating the code for this PR as part of our release process. It's harder (and more error-prone) for us to use `#{version}`. Do you think this suggestion is something we should do mandatorily?

Reply:
The reason to use `#{version}` instead of explicit version numbers is to minimize the number of individual lines to change on a version bump.

Are you generating a completely new formula for every release? A new version should only require changing the `url` and `sha256` stanzas, and your release process should not need to substitute for `#{version}` at all.